GoogleForCreators/web-stories-wp
# Dashboard: Changed Wellbeing template title's font-weight to light

## Summary

This PR changes the header/title font-weights from regular to light in the Wellbeing template.

## User-facing changes

- Page title/headers of the Wellbeing template have all been updated to a lighter font-weight.

## Testing Instructions

1.) Go to "Explore Templates"
2.) Click "view" for the Wellbeing template.
3.) Cycle through each page and see that the title/headers have a light (300) font-weight.
-- OR --
4.) Click "Use Template" and cycle through each page checking that the title/headers have a `light` font-weight.

---



Fixes #3194 

## Screenshots

The screenshots below show the before and after updates so you can see how the title/headers got a little be lighter.

**Before --> After**
<img src="https://user-images.githubusercontent.com/40646372/87710322-7872e080-c75a-11ea-8355-e0859c5eed90.png" height="300"><img src="https://user-images.githubusercontent.com/40646372/87710297-70b33c00-c75a-11ea-92d7-b58605f5500d.png" height="300">

**Before --> After**
<img src="https://user-images.githubusercontent.com/40646372/87710597-cd165b80-c75a-11ea-9c4f-685c72698e23.png" height="300"><img src="https://user-images.githubusercontent.com/40646372/87710352-81fc4880-c75a-11ea-8cd7-241393ac916f.png" height="300">

**Before --> After**
<img src="https://user-images.githubusercontent.com/40646372/87710630-d4d60000-c75a-11ea-9e88-92e22d211641.png" height="300"><img src="https://user-images.githubusercontent.com/40646372/87710377-8c1e4700-c75a-11ea-92b9-46ef65085eb3.png" height="300">

**Before --> After**
<img src="https://user-images.githubusercontent.com/40646372/87710651-dbfd0e00-c75a-11ea-9bf5-5bde16e07b37.png" height="300"><img src="https://user-images.githubusercontent.com/40646372/87710399-93455500-c75a-11ea-9688-df66fa1cc6d2.png" height="300">

**Before --> After**
<img src="https://user-images.githubusercontent.com/40646372/87710675-e3241c00-c75a-11ea-9eb0-3e5b219b372c.png" height="300"><img src="https://user-images.githubusercontent.com/40646372/87710455-a35d3480-c75a-11ea-86dd-9dbbf8a7c525.png" height="300">

**Before --> After**
<img src="https://user-images.githubusercontent.com/40646372/87710695-eb7c5700-c75a-11ea-8815-d2fab6bbcda7.png" height="300"><img src="https://user-images.githubusercontent.com/40646372/87710475-aa844280-c75a-11ea-99d1-e533e1b2d37a.png" height="300">

**Before --> After**
<img src="https://user-images.githubusercontent.com/40646372/87710711-f20ace80-c75a-11ea-9166-751729007c06.png" height="300"><img src="https://user-images.githubusercontent.com/40646372/87710493-b07a2380-c75a-11ea-94da-8640384ad4a2.png" height="300">

**Before --> After**
<img src="https://user-images.githubusercontent.com/40646372/87710724-fa630980-c75a-11ea-874b-2ab8b532288a.png" height="300"><img src="https://user-images.githubusercontent.com/40646372/87711373-08655a00-c75c-11ea-9e10-ddd6dd1df2d3.png" height="300">

**Before --> After**
<img src="https://user-images.githubusercontent.com/40646372/87710736-0058ea80-c75b-11ea-8362-9445a98c24ea.png" height="300"><img src="https://user-images.githubusercontent.com/40646372/87710541-bc65e580-c75a-11ea-9bf4-43334cd0f1b6.png" height="300">

**Before --> After**
<img src="https://user-images.githubusercontent.com/40646372/87710750-064ecb80-c75b-11ea-999c-a0d758296ce8.png" height="300"><img src="https://user-images.githubusercontent.com/40646372/87710572-c4be2080-c75a-11ea-996e-9965e9327718.png" height="300">
